automation/module_eval/src/types/index.ts

```diff
@@ -0,0 +1,186 @@
+/**
+ * Evaluation status for a criteria
+ *
+ * FRAMEWORK USAGE NOTES:
+ * - PASS: Criterion automatically verified as meeting requirements
+ * - FAIL: Criterion automatically verified as NOT meeting requirements
+ * - MANUAL: Requires human review (current default for most stub implementations)
+ *
+ */
+export enum EvaluationStatus {
+  PASS = 'pass',
+  FAIL = 'fail',
+  MANUAL = 'manual'
```

Comment:
Currently most of the criteria report back MANUAL, but some of these are because they are stubs for future automation, and some because they will (presumably) always be manual.  I wonder if we should split those two cases for clarity on what we intend.  Although on the other hand, deciding now whether something can (or should) be automated in the future may be impossible.